documentation/source/directives.rst

```diff
@@ -208,6 +210,8 @@ and additionally the ``content-only``, ``members``, ``protected-members``,
       :content-only:
       :outline:
       :members:
+      :membergroups: [...]
+      :members-only:
```

Comment:
Based upon some quick grep I believe the new options will only work for class, struct and interface directives. Are you sure it also works for group?

```console
$ grep -rIin _DoxygenClassLikeDirective -A1
breathe/directives.py:241:class _DoxygenClassLikeDirective(BaseDirective):
breathe/directives.py-242-    required_arguments = 1
--
breathe/directives.py:294:class DoxygenClassDirective(_DoxygenClassLikeDirective):
breathe/directives.py-295-    kind = "class"
--
breathe/directives.py:298:class DoxygenStructDirective(_DoxygenClassLikeDirective):
breathe/directives.py-299-    kind = "struct"
--
breathe/directives.py:302:class DoxygenInterfaceDirective(_DoxygenClassLikeDirective):
breathe/directives.py-303-    kind = "interface"
```

Reply:
This is about member groups, not doxygen groups

Reply:
I mean that the documentation you edit here is for the Breathe `doxygengroup` directive, which based upon the grep I believe doesn't take the new options.

Reply:
Whoops, you're right, my bad.